cabal-install/Distribution/Client/ProjectConfig.hs

```diff
@@ -362,9 +363,10 @@ findProjectRoot = do
 readProjectConfig :: Verbosity -> FilePath -> Rebuild ProjectConfig
 readProjectConfig verbosity projectRootDir = do
     global <- readGlobalConfig verbosity
-    local  <- readProjectLocalConfig      verbosity projectRootDir
-    extra  <- readProjectLocalExtraConfig verbosity projectRootDir
-    return (global <> local <> extra)
+    local  <- readProjectLocalConfig       verbosity projectRootDir
+    freeze <- readProjectLocalFreezeConfig verbosity projectRootDir
```

Comment:
I'm not sure if any of the `new-*` commands is supposed to modify `project.config`. I always assumed that `project.config` is a purely manually edited file, as opposed to`cabal.project.*` of which `cabal.project.local` and `cabal.project.freeze` are instances...